.pre-commit-config.yaml

```diff
@@ -0,0 +1,17 @@
+repos:
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: "v2.6.2"
+    hooks:
+      - id: prettier
+  - repo: https://github.com/pre-commit/mirrors-eslint
```

Comment:
Same as above.

Reply:
Replied in https://github.com/threshold-network/subgraph/pull/14#discussion_r889897415